resources/sdk/clisdkclient/templates/root.mustache

```diff
@@ -12,16 +12,29 @@ import (
 	"flag"
 	"fmt"
 	"strings"
+	"encoding/json"
+    "io/ioutil"
```

Comment:
There's a formatting issue here. Also in other parts of the file. Have a look at some of the other code added below.
The issue is probably due to a mix of tabs and spaces. It can look fine from VSCode but the spacing is different here so it shows up.